app/src/debug/res/raw/initial_experiments.json

```diff
@@ -81,6 +81,54 @@
           "proposedEnrollment": 7,
           "userFacingDescription": "Experiment to test the home screen configurations",
           "last_modified": 1621443780172
+      },
+      {
+          "slug": "feature-search-term-groups-android",
+          "appId": "org.mozilla.fenix",
+          "appName": "fenix",
+          "channel": "nightly",
+          "branches": [
+              {
+                  "slug": "enabled",
+                  "ratio": 0,
+                  "feature": {
+                      "enabled": true,
+                      "featureId": "search-term-groups"
+                  }
+              },
+              {
+                  "slug": "disabled",
+                  "ratio": 100,
+                  "feature": {
+                      "enabled": false,
+                      "featureId": "search-term-groups"
+                  }
+              }
+          ],
+          "outcomes": [],
+          "arguments": {},
+          "probeSets": [],
+          "startDate": null,
+          "targeting": "true",
+          "featureIds": [
+              "search-term-groups"
+          ],
+          "application": "org.mozilla.firefox_beta",
+          "bucketConfig": {
+              "count": 0,
+              "start": 0,
+              "total": 10000,
```

Comment:
This bucket configuration will target no-users, is that intended? If this is just for the purpose of using the Nimbus debugging tools to "opt-in" to this experiment, then it should be fine.